packages/cli/lib/prompts/createProjectPrompt.js

```diff
@@ -48,11 +48,18 @@ const createTemplateOptions = async (templateSource, githubRef) => {
   return config[PROJECT_COMPONENT_TYPES.PROJECTS];
 };
 
-const createProjectPrompt = async (githubRef, promptOptions = {}) => {
-  const projectTemplates = await createTemplateOptions(
-    promptOptions.templateSource,
-    githubRef
-  );
+const createProjectPrompt = async (
+  githubRef,
+  promptOptions = {},
+  templatesRequired = true
```

Comment:
I'm a little confused by what this is doing. Can you explain it?

Reply:
When we call the `createProjectPrompt` in the `migrate-app` command, we don't need the `template` option to appear in the prompts. This boolean allows us to hide that.

Reply:
Oh I see, thanks. This might be a little clearer with a name like `skipTemplatePrompt`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/0930b7255c8ed88d0f7e21420974c9223016f4e4